src/ResponseMerger/NuxtResponseMerger.php

```diff
@@ -29,8 +29,8 @@ public function mergeResponses(ResponseInterface $backendResponse, ResponseInter
 
     // Finally, merge responses and serve them.
     $page = $frontendResponse->getBody()->__toString();
-    $page = preg_replace('/<title(.*)>*<\/title>/', '<title$1>' . strip_tags($data['title']) . '</title>', $page);
-    $page = preg_replace('/<main role="main"(.*)><\/main>/', '<main role="main"$1>' . $data['content'] . '</main>', $page);
+    $page = preg_replace('/<title((?:\s*[a-zA-Z\-0-9]+(?:="[^"]*")*\s*)*)>[^<]*<\/title>/', '<title$1>' . strip_tags($data['title']) . '</title>', $page);
+    $page = preg_replace('/<main role="main"((?:\s*[a-zA-Z\-0-9]+(?:="[^"]*")*\s*)*)><!----><\/main>/', '<main role="main"$1>' . $data['content'] . '</main>', $page);
```

Comment:
This makes it now require a comment, what sucks:
> <!---->
the comment must be optional, we really should not require something optional like a comment

Reply:
hmm ok I thought this was static.. np I will just use the solution as for title